cmd/list.go

```diff
@@ -51,9 +53,64 @@ func RunList() *cobra.Command {
 				os.Exit(1)
 			}
 
-			fmt.Println(torrents)
+			PrintList(torrents)
 		}
 	}
 
 	return command
 }
+
+func PrintList(torrents []qbittorrent.Torrent) {
+	for _, torrent := range torrents {
+		fmt.Printf("%-60s\t[%s]\n", torrent.Name, torrent.State)
+
+		fmt.Printf(
+			"Downloaded: %s / %s%10s",
+			BytesToHumanReadable(float64(torrent.Completed)),
+			BytesToHumanReadable(float64(torrent.TotalSize)),
+			"",
+		)
+
+		if torrent.DlSpeed > 0 {
+			fmt.Printf(
+				"DL Speed: %s/s%10s\t",
+				BytesToHumanReadable(float64(torrent.DlSpeed)),
+				"",
+			)
+		} else if torrent.UpSpeed > 0 {
+			fmt.Printf(
+				"UP Speed: %s/s%10s\t",
+				BytesToHumanReadable(float64(torrent.UpSpeed)),
+				"",
+			)
+		} else {
+			fmt.Printf("%27s\t", "")
+		}
+
+		hours := (torrent.TimeActive / 60) / 60
+		minutes := (torrent.TimeActive / 60) - (hours * 60)
+		fmt.Printf("Time Active: %dh%dm\n", hours, minutes)
+
+		fmt.Printf("Save Path: %s\n", torrent.SavePath)
+		fmt.Println()
+	}
+}
+
+func BytesToHumanReadable(size float64) string {
```

Comment:
Make lower case since it's no reused. 

And nice doing all the things yourself, but I often reach for [go-humanize](https://github.com/dustin/go-humanize) which have some other good utilities. No need to change this :slightly_smiling_face: